### PR TITLE
Group and user page buttons

### PR DESCRIPTION
**Pull request by xxyzz**
- Add message button to group info page
- Change leave group and delete friend button to menu button.
- Add warning color

### CODE REVIEW

File: src/app/Views/user.html [hunk 1]
```diff
@@ -31,13 +37,12 @@
...
+        <i class="fa fa-trash warning"></i>
     </div>
-    <div class="text" i18n="@@Delete">Delete</div>
+    <div class="text warning" i18n="@@Delete">Delete</div>
 </div>
 <div class="menu-holder"></div>
 
 <div class="buttons">
     <button class="success" *ngIf="info && !areFriends" (click)="request(info.id)" i18n="@@SendFriendRequest">Send Friend Request</button>
-    <button class="success" *ngIf="info && areFriends" (click)="talk(conversationId)" i18n="@@Message">Message</button>
```
Comment (Anduin2017) [inline src/app/Views/user.html:42]: Why do you remove this button?

File: src/app/Views/user.html [hunk 2]
```diff
@@ -11,6 +11,12 @@
 </div>
 
 <div class="menu-holder"></div>
+<div class="menu-button" *ngIf="info && areFriends" (click)="talk(conversationId)">
```
Comment (Anduin2017) [inline src/app/Views/user.html:14]: Any reason to put it here?

File: src/app/Views/group.html [hunk 3]
```diff
@@ -22,7 +28,9 @@
     </div>
     <div class="text" i18n="@@Report">Report</div>
 </div>
-
-<div class="buttons">
-    <button class="warning" (click)="leaveGroup(conversation.displayName)" i18n="@@LeaveGroup">Leave Group</button>
+<div class="menu-button" (click)="leaveGroup(conversation.displayName)">
```
Comment (Anduin2017) [inline src/app/Views/group.html:31]: I will provide you more APIs about group management.

**Review by Anduin2017 — APPROVED** [review]

File: src/app/Views/user.html (hunk 2 above)
Comment (xxyzz) [reply src/app/Views/user.html:14]: I think the message button is more commonly used than the delete button, so it should be moved to the top. And delete button usually located at the bottom.

Comment (Anduin2017) [reply src/app/Views/user.html:14]: Really great idea!

**Pull request merged**